### ISSUE TITLE
Support for Java language

### ISSUE THREAD

**Issue by zabil** [getgauge/gauge-vscode#83]
### Expected behavior
Support features that currently works only on javascript.

* Goto definition
* Find step references
* Generate implementation stub
* etc

**Comment by v-mwalk** [comment]
+1

**Comment by sguptatw** [comment]
Blocker - The latest nightly available is `Nightly/0.7.0.nightly-2018-11-28`

**Comment by Debashis9012** [comment]
Tested the above features its working as excepted.
Tested with below version

```
Gauge version: 1.0.4.nightly-2019-01-22
Commit Hash: 3a9a647

Plugins
-------
html-report (4.0.7.nightly-2019-01-11)
java (0.7.0.nightly-2019-01-24)
js (2.3.4.nightly-2018-12-20)
```
